src/Analyser/MutatingScope.php

```diff
@@ -2638,6 +2638,7 @@ public function enterFunction(
 		?bool $isPure = null,
 		bool $acceptsNamedArguments = true,
 		?Assertions $asserts = null,
+		?string $phpDocComment = null,
```

Comment:
1) All other parameters are specific about what they do here, I'd expect your added parameter to be specific about `@param-out`. Handling over the whole `$phpDocComment` smells like bad code.
2) In any case, you need to do the same for enterClassMethod.

Reply:
actually its a leftover and will be dropped